tests/cypress/e2e/unit_tests/upgrade_fleet.spec.ts

```diff
@@ -133,4 +132,54 @@ if (!/\/2\.8/.test(Cypress.env('rancher_version')) && !/\/2\.9/.test(Cypress.env
       })
     );
   });
-};
+}
+
+describe('Test "fleet-agent" image version on each downstream cluster',  { tags: '@upgrade' }, () => {
+  qase(163,
+    it('FLEET-163: Test "fleet-agent" image version on each downstream cluster after upgrade ', { tags: '@fleet-163' }, () => {
+      const dsAllClusterList = ['imported-0', 'imported-1', 'imported-2']
+      cy.log("===========================");
+      cy.log("Cluster Upgraded: " + upgrade);
+      cy.log("===========================");
+
+      //'fleet_app_version' is before upgrade version.
+      // For example 'fleet_app_version' is fleet-105.0.3+up0.11.3, after splitting
+      // we get 'fleet_agent_version' is '0.11.3'.
+      if (upgrade) {
+        const fleetAgentVersionBeforeUpgrade = Cypress.env('fleet_app_version').split('+up')[1];
+        cy.log('Fleet Agent image version BEFORE upgrade:' + fleetAgentVersionBeforeUpgrade)
+
+        // Check fleet-agent version of the downstream clusters.
+        dsAllClusterList.forEach((dsCluster) => {
+          cy.accesMenuSelection(dsCluster, 'Workloads', 'Pods');
+          cy.nameSpaceMenuToggle('All Namespaces');
+          cy.filterInSearchBox('fleet-agent');
+          cy.verifyTableRow(0, 'Running', 'fleet-agent-0');
+          cy.contains('fleet-agent-0').click();
+          cy.wait(500);
+          cy.clickButton('Config');
```

Comment:
Question: in this point you could also extract it from the screen where line 157 leaves you at with this locator: `span[class='formatter-pod-images'] span:nth-child(1)` (can be improved)
No issues in leaving it as it is, but if you want maybe you can save a few steps